### PR TITLE
NER new setup instructions documentation

### PR DESCRIPTION
**Pull request by harjinder7**
## JIRA Ticket Number

JIRA TICKET: ML-3625

## Description of change
Documentation for setting up NER in dev is added. Instructions are mention how to setup NER with help of docker and setup elastic-search for entities.

### CODE REVIEW

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Review by naseem-shaik — COMMENTED** [review]
Move the files outside the docker folder after verifying if there is any reason why we need to keep them inside and copy them out again. 

Review the documentation for minor errors. (link to notion etc)

1. Review and launch
2. Update the bot calibrator with the latest develop pull
3. Share with the engineering team  steps on how to update and redeploy if needed.

File: Dockerfile-python3 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ENV DJANGO_SETTINGS_MODULE=chatbot_ner.settings
+ENV PORT=8081
+ENV TIMEOUT=600
+ENV MAX_REQUESTS=1000
+# ENV ENVKEY=
+# ENV NEWRELIC_LICENSE_KEY=
+EXPOSE 8081
+ADD . /app
+RUN chmod +x /app/entrypoint.sh
+CMD /app/entrypoint.sh
```
Comment (naseem-shaik) [inline Dockerfile-python3:27]: Rename the entrypoint.sh file or confirm with Jaspreet that it won't cause name conflict on staging or production.

**Review by naseem-shaik — COMMENTED** [review]
Remove unrequired files from dockers folder, if nothing is needed inside the docker folder, remove it entirely.

File: Dockerfile-python3 (hunk 1 above)
Comment (harjinder7) [reply Dockerfile-python3:27]: Confirmed with Jaspreet, there won't be no conflict on staging and prod.

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

File: Dockerfile-python3 (hunk 1 above)
Comment (naseem-shaik) [reply Dockerfile-python3:27]: Okay.

**Review by naseem-shaik — APPROVED** [review]
LGTM. Please ensure that these steps are tested on dev setup once.

**Comment by harjinder7** [comment]
Unit test case pass during dev testing

image

**Review by HiteshC6595 — APPROVED** [review]
We have removed docker folder from NER setup. As per devops the NER setup file for Staging and prod are getting downloaded from S3 so there no need of these files anymore. Also we ran unit testcases and entity testing suite on these changes and it got passed successfully hence approving the PR

**Pull request merged**